### PR TITLE
Fixes #1825. Parent MenuItem stay focused if child MenuItem is empty.

### LINKED ISSUE #1825: Parent MenuItem loses focus if child MenuItem is empty
Happens by mouse and keyboard.

### PR DESCRIPTION
**Pull request by BDisp**

### CODE REVIEW

File: Terminal.Gui/Views/Menu.cs [hunk 1]
```diff
@@ -1296,11 +1299,11 @@ internal void Activate (int idx, int sIdx = -1, MenuBarItem subMenu = null)
 				previousFocused = SuperView == null ? Application.Current.Focused : SuperView.Focused;
 
 			OpenMenu (idx, sIdx, subMenu);
-			if (!SelectEnabledItem (openCurrentMenu.barItems.Children, openCurrentMenu.current, out openCurrentMenu.current)
-				&& subMenu == null && !CloseMenu (false)) {
+			//if (!SelectEnabledItem (openCurrentMenu.barItems.Children, openCurrentMenu.current, out openCurrentMenu.current)
```
Comment (tig) [inline Terminal.Gui/Views/Menu.cs:1302]: Why comment this out vs. just deleting it?

**Review by tig — COMMENTED** [review]
Why not just delete the old code vs. comment it out?

File: Terminal.Gui/Views/Menu.cs (hunk 1 above)
Comment (BDisp) [reply Terminal.Gui/Views/Menu.cs:1302]: Ok I'll delete it.

**Comment by BDisp** [comment]
Ok I'll delete them.

**Pull request merged**